apps/vscode-wing/src/extension.ts

```diff
@@ -67,9 +66,7 @@ async function startLanguageServer(context: ExtensionContext) {
     execSync(`chmod +x ${serverPath}`);
   }
 
-  const wingsdkManifestRoot = dirname(
-    require.resolve("@winglang/wingsdk/.jsii")
-  );
+  const wingsdkManifestRoot = context.asAbsolutePath("resources/wingsdk");
```

Comment:
Is this config/setup needed by all apps consuming the Wing LSP? I wonder if we could add this SDK-types-injection as a build step to the LSP instead so it's bundled with that code instead. (This would match up with how the Wing CLI bundles the compiler with the SDK today)

Reply:
Honestly, I've been thinking that we should embed the .jsii into wingc itself, it would greatly simplify the story for every dependency (vscode, lsp, playground) since it is always needed. 

I thought about this a long time ago and thought it was hacky, but as time passed it seems more and more reasonable.

Reply:
Makes sense to me. 👍  Wing SDK isn't a "modular" piece of the compiler / they are both needed for the compiler to work, so coupling them seems natural

Reply:
https://github.com/winglang/wing/issues/1083 👍